### PR TITLE
Adding the `ccache` and `clamav` packages.

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Adding the `ccache` and `clamav` packages as they're required by some of our partners.

###### Change Log  

- Added the `ccache` and `clamav` packages.

###### Does this affect the toolchain?  

No.

###### Test Methodology

- Local build.

### CODE REVIEW

File: SPECS/clamav/clamav.spec [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ClamAV® is an open source (GPL) anti-virus engine used in a variety of situations
+including email scanning, web scanning, and end point security. It provides a number
+of utilities including a flexible and scalable multi-threaded daemon, a command
+line scanner and an advanced tool for automatic database updates.
+
+%prep
+%setup -q
+
+%build
+./configure \
```
Comment (MateuszMalisz) [inline SPECS/clamav/clamav.spec:35]: consider using %configure instead to avoid passing some of the flags

File: SPECS/clamav/clamav.spec [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    --prefix=%{_prefix} \
+    --exec_prefix=/usr \
```
Comment (thcrain-msft) [inline SPECS/clamav/clamav.spec:37]: Preferably we would change `/usr` -> `%{_exec_prefix}` here

**Review by thcrain-msft — APPROVED** [review]
🚢 

LGTM other than the feedback on the configure line for clamav

File: SPECS/clamav/clamav.spec (hunk 2 above)
Comment (thcrain-msft) [reply SPECS/clamav/clamav.spec:37]: But if %configure is used instead, this won't be a problem

**Review by dmcilvaney — APPROVED** [review]

**Comment by dmcilvaney** [comment]
fyi, looks like the new spec linter has some feedback: https://github.com/microsoft/CBL-Mariner/actions/runs/322486365
Haven't looked personally at them. @thcrain-msft Looks like the linter didn't flag an error even though it found changes.

**Pull request merged**